src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return (
+    sourceString
+      // Breaks sourceString by semicolon into separate stings
+      .split(';')
+
+      // Breaks line into property and value and removes extra space charachters
+      .map((el) => el.split(':').map((x) => x.trim()))
+
+      // Leaves only valid elements
+      .filter((el) => el.length === 2)
```

Comment:
The filter condition is correctly checking for the length to be 2, which implies that both the property and value exist in the CSS declaration. However, it would be more robust to also check that both elements are not empty strings.